ztp/resource-generator/Containerfile

```diff
@@ -36,6 +42,11 @@ RUN chown -R 1001:1001 $ZTP_HOME
 # Copy in the entrypoint scripts
 COPY --from=builder $BUILDER_ZTP/resource-generator/entrypoints/* /usr/bin
 COPY --chown=1001 --from=builder $BUILDER_ZTP/resource-generator/exportkustomize.sh /
+# Copy pgt2acm script
+COPY --from=builder $BUILDER_ZTP/tools/pgt2acm/pgt2acm /usr/bin
+COPY --from=builder /usr/bin/tree /usr/bin
+COPY --from=builder  $BUILDER_ZTP/tools/pgt2acm/kustomize /kustomize-pgt2acm
```

Comment:
nit: extra space between `--from=builder` and `$BUILDER_ZTP` on line 48 only

Reply:
fixed